### DISCUSSION TITLE
Repolarin Gorunurlugu

### DISCUSSION THREAD

**Discussion by guvenrdvan** [gusanmaz/BMB212-Algorithms-2223#11] [General]
Davet linki ile oluşturduğumuz repolar private olarak gözüküyor. Sizin reponuz içinde oluşturduğumuz için görebiliyor musunuz yoksa public e çekebiliyor muyuz?

**Comment by gusanmaz** [comment]
Ben gorebiliyorum nku-bm-gu organizayonu altinda olusturulan repolari. Bunun karari Github Classroom altinda odev tanimlarken verildigi icin sizin reponun erisim kapsamini degistirme seceneginiz olmuyor bildigim kadariyla. Odevlerde erisim kapsaminin public belirlenmesi durumunda haberiniz bile olmadan yazdiginiz kodlar baskalari tarafindan kullanilabilir. Odevlerde kopya cekilmesi konusunda bir endisem olmasa tum odevler icin erisim kapsaminin public olmasini tercih ederdim.
![image](https://user-images.githubusercontent.com/2552975/228983174-22f93f27-13fb-4f1c-854d-358ee1e7cbbb.png)

**Reply by guvenrdvan** [reply]
Tamamdır siz görüyorsanız sıkıntı yok.
